e2e/integration/suggest.test.js

```diff
@@ -100,3 +108,35 @@ it("lets the suggester assign a topic to the resource", () => {
 	cy.findByRole("heading", { level: 1 }).click();
 	cy.findByText("HTML/CSS").should("exist");
 });
+
+it("lets an admin user publish a resource", () => {
+	cy.visit("/");
+
+	cy.task("seed", { users: [admin] });
+	cy.logInAs("admin@codeyourfuture.io");
+	cy.visit("/");
```

Comment:
💡 It seems odd to visit the page, _then_ update the DB, then login, then _reload_ the home page (which you're on after logging in), maybe:
```suggestion
	cy.task("seed", { users: [admin] });
	cy.visit("/");
	cy.logInAs("admin@codeyourfuture.io");
```